src/main/java/JustinBot.java

```diff
@@ -0,0 +1,64 @@
+import justin.command.Command;
+import justin.DukeException;
+import justin.Parser;
+import justin.Storage;
+import justin.TaskList;
+import justin.Ui;
+
+/**
+ * Represents the main class in which
+ * the program is run.
+ * @author Justin Cheng.
+ */
+public class JustinBot {
+
+    private Storage storage;
+    private TaskList tasks;
+    private Ui ui;
+
+    /**
+     * Constructor for the Duke class.
+     * @param filePath The path of the data file
```

Comment:
I think an empty line between the description and the parameter sections should be better? Might help with readability, especially since your descriptions tend to be "wrapped" in a few lines. This applies for other javadoc comments too.